index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Document</title>
+</head>
+
+<body>
+  <main id="main">
+    <h1> NGOZA & MARGARET</h1>
+    <p id="title">Microverse Trials</p>
+    <figure id="img-div">
+      <img id="image" src="img.png">
+      <figcaption id="img-caption">Ngoza and Margaret working on a project</figcaption>
+    </figure>
+    <h2 id="tribute-info">Instructions</h2>
+    <ul>
+      <li>The Microverse Trials is one of the most important parts of the application process.</li>
+      <br>
+      <li>You will be paired with three (3) other applicants and will meet with each of them separately to build three
+        (3) different programming projects.</li>
+      <br>
+
+      <li>You will spend four (4) consecutive hours working with each of your partners, which means 12 hours of
+        collaborative work under your belt!</li>
+      <br>
+
+      <li>Each session with a new coding partner will be done synchronously, using pair programming, on a video call
+      </li>
+    </ul>

```

Comment:
Please use htm semantic tags always. you can wrape this code around the article tag

Reply:
- [ ] It is important that for every comment you add you should as well go deep by explaining the `why` a change is being requested. This would assist the students to better understand the reason for a change and they know how best to go about it in their next projects :+1: :sparkles: